### PR TITLE
Add meshery-nighthawk helm chart

### LINKED ISSUE #10370: [DevOps] Create new Helm sub-chart for Meshery Adapter for Nighthawk
### Current Behavior
A new Meshery Adapter is being created in the v0.8 release: https://github.com/meshery/meshery-nighthawk

### Desired Behavior
A new Helm chart similar to the other adapter home charts will need to be created for this new adapter: https://github.com/meshery/meshery/tree/master/install/kubernetes/helm/meshery/charts

### Implementation
1. Duplicate one of the existing adapter charts.
2. Replace all adapter specific references with references to `meshery-nighthawk` adapter.

Note: this adapter runs on port 10013/tcp by default.

### Acceptance Tests
1. `helm install meshery meshery/meshery -n meshery` deploys an instance of this adapter.

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)
- 🧪 [Meshery Test Plan Spreadsheet](https://docs.google.com/spreadsheets/d/13Ir4gfaKoAX9r8qYjAFFl_U9ntke4X5ndREY1T7bnVs/edit#gid=0)

### PR DESCRIPTION
**Pull request by lekaf974**
**Notes for Reviewers**

This PR fixes #10370 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by lekaf974** [comment]
Hello, my first contribution waiting for any feedback

**Comment by leecalcote** [comment]
Thank you, @lekaf974 👍

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+name: meshery-nighthawk
+description: Meshery Adapter for App Mesh chart.
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml:3]: Please also update the description for meshery-nighthawk.

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+icon: https://meshery.io/images/logos/meshery-logo.png
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml:4]: We have the meshery-nighthawk icon: https://github.com/meshery/meshery-nighthawk/blob/master/assets/img/icon/nighthawk-logo.png, can we use that? // @leecalcote

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+source:
+  - https://github.com/layer5io/meshery-nighthawk
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml:6]:
```suggestion
  - https://github.com/meshery/meshery-nighthawk
```

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.7.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+# TODO: Need the confirm the image tag.
+appVersion: stable-latest
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml:27]:
```suggestion
appVersion: edge-latest
```
We only have the edge-latest version of meshery-nighthawk image currently: https://hub.docker.com/layers/meshery/meshery-nighthawk/edge-latest/images/sha256-4efc9a803b2c31b0395596af6bc4b1289da067fc058684cbf49799eb56e74901?context=explore

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/values.yaml [hunk 5]
```diff
@@ -0,0 +1,85 @@
+# Default values for meshery-nighthawk.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: layer5/meshery-nighthawk:stable-latest
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/values.yaml:8]:
```suggestion
  repository: meshery/meshery-nighthawk:edge-latest
```

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# meshery-nighthawk
+
+![Version: 0.5.0](https://img.shields.io/badge/Version-0.5.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: stable-latest](https://img.shields.io/badge/AppVersion-stable--latest-informational?style=flat-square)
+
+Meshery Adapter for App Mesh chart.
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.md:5]: Ditto

File: install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+# meshery-nighthawk
+
+![Version: 0.5.0](https://img.shields.io/badge/Version-0.5.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: stable-latest](https://img.shields.io/badge/AppVersion-stable--latest-informational?style=flat-square)
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/charts/meshery-nighthawk/README.md:3]: The "Version" and "AppVersion" should also be updated.

File: install/mesheryapp.dockerapp/docker-compose.yml [hunk 8]
```diff
@@ -30,14 +30,21 @@ services:
...
+    x-enabled: ${consul.enabled}
   meshery-nsm:
     image: layer5/meshery-nsm:stable-latest
     labels:
       - ${labels.watchtower}
     ports:
       - "${nsm.port}:10004"
     x-enabled: ${nsm.enabled}
+  meshery-nighthawk:
+    image: layer5/meshery-nighthawk:stable-latest
```
Comment (gyohuangxin) [inline install/mesheryapp.dockerapp/docker-compose.yml:42]:
```suggestion
    image: meshery/meshery-nighthawk:edge-latest
```

**Review by gyohuangxin — CHANGES_REQUESTED** [review]
@lekaf974 Great! Thank you for contributing on this, I left some comments.

**Comment by lekaf974** [comment]
@gyohuangxin thx for feedback, will do the changes based on comments soon

File: install/kubernetes/helm/meshery/Chart.yaml [hunk 9]
```diff
@@ -68,6 +69,9 @@ dependencies:
   - name: meshery-perf
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/Chart.yaml:69]: Do we need to delete the meshery-perf component? @leecalcote

File: install/kubernetes/helm/meshery/values.yaml [hunk 10]
```diff
@@ -156,6 +156,11 @@ meshery-perf:
   fullnameOverride: meshery-perf
```
Comment (gyohuangxin) [inline install/kubernetes/helm/meshery/values.yaml:156]: Ditto.

**Comment by leecalcote** [comment]
@lekaf974 thanks for digging in here.

Are you all done incorporating feedback and ready for another review?

**Comment by lekaf974** [comment]
> @lekaf974 thanks for digging in here.
> 
> Are you all done incorporating feedback and ready for another review?

Yes all have been updated and ready for new feedback

**Review by gyohuangxin — APPROVED** [review]

**Comment by leecalcote** [comment]
> > @lekaf974 thanks for digging in here.
> > Are you all done incorporating feedback and ready for another review?
> 
> @leecalcote Yes all have been updated and ready for new review

👨‍🎤 👍

**Pull request merged**